fdbclient/NativeAPI.actor.cpp

```diff
@@ -775,9 +779,51 @@ Future<Void> DatabaseContext::onConnected() {
 	return cluster->onConnected();
 }
 
+ACTOR static Future<Void> changeConnectionFileImpl(Reference<ClusterConnectionFile> connFile, DatabaseContext* self) {
+	TEST(true); // Change connection file
+	TraceEvent("ChangeClusterFile")
```

Comment:
This trace event name (and maybe the idea of "changing" the connection file in general) might be confused with changing the connection string for a particular cluster, such as by changing the coordinators. I think it may be more clear to describe this as something like changing your cluster rather than the file to avoid the ambiguity. Or maybe a different verb (like switch?) communicates the concept more clearly.

Reply:
Makes sense, I think switch communicates the concept more clearly. Done.